omegatrade-app/backend/app/controller/user.controller.js

```diff
@@ -11,59 +12,30 @@ const { v4: uuidv4 } = require('uuid');
  * @param {JSON} req contains request headers and payload businessEmail,fullName,password,photoUrl,provider
  */
 exports.register = async function (req, res) {
-    const body = req.body;
-    await User.findUser(body.businessEmail, function (err, data) {
-        if (data && data.userId) {
-            res.status(400).json({
-                success: false,
-                message: 'Email Already Exists',
-            });
-        }
-        if (err) {
-            res.status(400).json({
-                success: false,
-                message: 'Something went wrong while fetching user!'
-            });
-        }
-    });
-    const salt = await bcrypt.genSalt(10);
-    const password = await bcrypt.hash(body.password, salt);
-    delete body.password; 
-    delete body.confirmPassword;
-    const user = {
-        ...body,
-        userId: uuidv4(),
-    }
     try {
-        await User.registerUser({ ...user,password }, function (err, data) {
+        const body = req.body;
+        const [registerUser] = await User.findUser(body.businessEmail);
+        if (registerUser && registerUser.userId) {
+            return res.status(400).json({ success: false, message: 'Registration Failed, Email Already Exists!' });
```

Comment:
nit: Don't capitalize each word of the error message.

Reply:
the above issue has been resolved.